### DISCUSSION TITLE
About Query

### DISCUSSION THREAD

**Discussion by c1kk** [ravendb/ravendb#13386] [Q&A] [answered]
I have a question about query, RavenDB can query from 2 database in 1 query ?  Example : DB 1 have collection customer 1 , DB 2 have collection customer 2 . Can i use 1 query to select all customer from 2 DB ?

**Comment by DejanMilicic** [comment, marked as answer]
A session is always opened against a single database.
This database can either be a default one used for all sessions, or you can specify it explicitly
https://ravendb.net/docs/article-page/5.3/Csharp/client-api/setting-up-default-database

So, to query from 2 databases, you will have to open two sessions and execute queries, then combine results you got from both queries
